mobile/test/swift/integration/DirectResponseContainsHeadersMatchIntegrationTest.swift

```diff
@@ -20,6 +20,7 @@ final class DirectResponseContainsHeadersMatchIntegrationTest: XCTestCase {
     .build()
 
     let engine = TestEngineBuilder()
+      .useLegacyBuilder(true)
```

Comment:
just for my own awareness, are these tests not cleaned up yet, or are you just trying to ensure coverage for old and new?

Reply:
The C++ builders don't support direct responses (yet).

I added support for that locally, and I should be able to put up a PR for discussion today or tomorrow.

Reply:
https://github.com/envoyproxy/envoy/pull/25552